app/messaging/process-backoffice.js

```diff
@@ -0,0 +1,31 @@
+const util = require('util')
+const { getAll, getApplicationCount } = require('../repositories/application-repository')
+const { backOfficeResponseMsgType, backOfficeResponseQueue } = require('../config')
+const sendMessage = require('../messaging/send-message')
+const states = require('./states')
+
+const processBackOfficeRequest = async (msg) => {
+  const responseMessage = msg.body
+  const { sessionId } = msg
+  try {
+    console.log('BackOffice Request received:', util.inspect(msg.body, false, null, true))
+    // Get ID
+    const result = await getAll(msg.body.limit ?? 10, msg.body.offset ?? 0, msg.body.search.text)
```

Comment:
I think you'd be better off setting the defaults for the query in the `getAll` function rather than at the callee.